force_wfmanager/central_pane/result_table.py

```diff
@@ -62,8 +49,25 @@ def _get_columns(self):
                 for index, name
                 in enumerate(self.analysis_model.value_names)]
 
-    # Response to model change
+    # ----
+    # View
+    # ----
+    def traits_view(self):
```

Comment:
```suggestion
    def default_traits_view(self):
```

`traits_view()` has a number of functions. The method to implement to provide a default `View` programmatically is `default_traits_view()`.

Reply:
Fixed, thanks.